Classes/Domain/Model/Address.php

```diff
@@ -214,6 +222,27 @@ public function __construct()
         $this->image = new ObjectStorage();
     }
 
+    /**
+     * sets the hidden attribute
+     *
+     * @param boolean $hidden
```

Comment:
needs to be named `bool` instead of `boolean`

Reply:
thx, changed